system-parachains/collectives/collectives-polkadot/src/ambassador/migration.rs

```diff
@@ -0,0 +1,388 @@
+			(3, hex_literal::hex!("1035801fd00144e10a3933ed859f8236bbffb93a7ac515bab9f1ca53cbb3f776")),
+			(3, hex_literal::hex!("28d50241999da5b300f01f3004a67a25a11854608f1f437ab86ed2e115243a43")),
+			(3, hex_literal::hex!("aa48a9775be7af0521acefce054dc7e9e461814dc167a5cabf52aef8534d8249")),
+			(3, hex_literal::hex!("0ef2cc1000f878a3880a09d698b5375f20c4ab3d8b3a1b783c8150faca3da65a")),
+			(3, hex_literal::hex!("e86b14052f27742916a13482c39afc8d9a8f873d799c9aa070bbd045570baa66")),
+			(3, hex_literal::hex!("742f10a2b57e5ec3247c0ae6da2a2fdb4a731324dc7a2edfe0f4fc761e1a4d3f")),
+			(3, hex_literal::hex!("4c2ded7ca2dd19095123de090a46149c2047d0aa4c6ce195490563d881f7491b")),
+			(3, hex_literal::hex!("542aede1ac86f5f0326efb4b2edde2b0ccd56635d8d76e9ae86d9d93f8d02063")),
+			(3, hex_literal::hex!("a0e1c76edd8151d4841665925d01a029fe176eecc24297bf931d594502293159")),
+			(3, hex_literal::hex!("f8ff75032359f0de13264a134c3e88089cf6a2a31e5cf3cdfe405a4e272f0508")),
+			(3, hex_literal::hex!("aea3ca653928298cd4d1d64cf916aacf72e4e2ca435453941a73327a8dd0a00b")),
+			(3, hex_literal::hex!("a6796c4e02fca3ded862fce7bab207c1eadac8af1028fcb9e5bf4a0fa3aaae29")),
+			(3, hex_literal::hex!("1cbf2d072567bdfeb00359e9d318e7b425a65449eb94b1a8f5ca0c28a9513878")),
+			(3, hex_literal::hex!("c07b9222722b0cf30cd9034487a054a00ff3977ac034a7c05caa1b1ac45c8e73")),
+			(3, hex_literal::hex!("0cd1574cf7fc649a8571d82427b42732e83bcd62f0e3192e92f202ca06f0d53c")),
+			(3, hex_literal::hex!("0c691601793de060491dab143dfae19f5f6413d4ce4c363637e5ceacb2836a4e")),
+			(3, hex_literal::hex!("2e5157fa386365bb2a60fc6f415bf070acdae47078be1396e3dc47ccb837d459")),
+			(3, hex_literal::hex!("58efadc57a1952fc5829948986e5b86e2b7873ee16510800628e8bfd0344ac5a")),
+			(3, hex_literal::hex!("86c3585c906e4928f030b4735d375cee0410db104908788133281b53533b5633")),
+			(3, hex_literal::hex!("2055808c210d863dfc372ec85beafa8fd3a8ff497f8eaee401ef05bf27d3065b")),
+			(3, hex_literal::hex!("da92c32ab2b4e1a46bb659cb6fd22fc824611e4c2803fbedd93c246f97c67118")),
+			(3, hex_literal::hex!("6ae93e7162785a77d3a2c0413a9ee04af1b948ba5df9ac191552b72e1dd49b71")),
+			(4, hex_literal::hex!("5c23daa77af39166a326377a006496753620093e13210a8567abed59e340eb06")),
+			(4, hex_literal::hex!("d6b8ec23dc68f20b5d315007d9c1a6706f9bd5c883319181129e76a89e978155")),
+			(4, hex_literal::hex!("00213c9e2131a7e4b10266e379a06774bf4a6a5e9f5632f31cb16bebd2cfb644")),
+			(4, hex_literal::hex!("6a7cfc5b4a583e6ebeddeb24fcb17ee5cbfd5357bb41effa87adbe2a9e46fb41")),
+		];
+	}
+
+	/// Implements `OnRuntimeUpgrade` trait.
+	#[allow(dead_code)]
+	pub struct Migration<T, I = ()>(PhantomData<(T, I)>);
+
+	impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I>
+	where
+		<T as frame_system::Config>::AccountId: From<[u8; 32]>,
+	{
+		#[cfg(feature = "try-runtime")]
+		fn pre_upgrade() -> Result<Vec<u8>, sp_runtime::TryRuntimeError> {
+			let has_members = <Members<T, I>>::iter().next().is_some();
```

Comment:
```suggestion
			let has_members = <Members<T, I>>::iter_keys().next().is_some();
```
Just in case of undecodable values.